### DISCUSSION TITLE
How to create cutting planes and crop a subdomain?

### DISCUSSION THREAD

**Discussion by nishant-mistry** [neperfepx/neper#480] [Q&A] [answered]
Hi, 

I used to generate periodic RVEs with flat faces using Voro++ as follows:
1. randomly seeding a unit cube 
2. copying these seeds to unit cubes all around the initial cube
3. asking voro++ to create periodic tessellations with respect to all these seed points and finally,
4. cutting out the central unit cell by defining 8 planes of the cube

Is this possible to do it with Neper? I wish to use this geometry with tet meshing with abaqus that means I must have the file in scalar format (.tess).

**Comment by rquey** [comment, marked as answer]
No. Neper generates periodic tessellations where grains are not cut.  The resulting mesh can be used in Abaqus, although the boundary conditions must probably be applied differently.

Your steps 1-4 might work, but the generated mesh will not be periodic.
